src/electron/Bootstrap.ts

```diff
@@ -0,0 +1,146 @@
+import { app, ipcMain, Menu, MenuItem, nativeImage, Tray } from "electron"
+import log from 'electron-log'
```

Comment:
inconsistent quote characters. Surprised linter didn't yell at you for this.

Reply:
same.